ghmirror/core/mirror_requests.py

```diff
@@ -118,63 +173,26 @@ def online_request(method, url, auth, data=None, url_params=None):
         if last_mod is not None:
             headers['If-Modified-Since'] = last_mod
 
-    resp = requests.request(method=method,
-                            url=url,
-                            headers=headers,
-                            timeout=REQUESTS_TIMEOUT,
-                            params=parameters)
+    resp, valid_response = _online_request(method=method,
+                                           url=url,
+                                           headers=headers,
+                                           data=data,
```

Comment:
If I understand correctly the code, you are only getting here for `GET` requests. Why do you need to pass `data`?

Reply:
The original request was passing the data, and I replicated the same in the wrapper, but maybe does not make sense and we can remove it

Reply:
Are you sure about that? I don't see it here: https://github.com/app-sre/github-mirror/blob/master/ghmirror/core/mirror_requests.py#L121-L125